ui/summit-2024/src/pages/Home/components/Hero.tsx

```diff
@@ -62,7 +64,14 @@ const Hero = () => {
               },
             }}
           >
-            Voting Closes {formatISODate(eventCache.eventEndDate)}
+            {eventCache.notStarted
+              ? "Voting Opens " + formatISODate(eventCache.eventStartDate)
+              : eventCache.active
+                ? "Voting Closes " + formatISODate(eventCache.eventEndDate)
+                : !eventCache.proposalsReveal
+                  ? "Results Announced " +
+                    formatISODate(eventCache.proposalsRevealDate)
+                  : "THE RESULTS ARE IN!!!"}
```

Comment:
Is it actually meant to be "THE RESULTS ARE IN!!!"?

Reply:
It also sounds strange to me but I got it from the [Figma designs](https://www.figma.com/design/cRGPl1mAbAsY9MUuJM14xf/Cardano-Ballot-2024?node-id=1222-5339&t=5I72gL5oWCUqRzPp-4)